docs/onderzoek-bekijken/7-beslisboom/utrecht-keuzehulp-geldhulp-beslisboom-2.md

```diff
@@ -5,6 +5,7 @@ hide_table_of_contents: false
 sidebar_label: Gebruikersonderzoek keuzehulp geldhulp bij kind laten sporten - deel 2
 pagination_label: Gebruikersonderzoek keuzehulp geldhulp bij kind laten sporten - deel 2
 description: Verbeterde versie van een digitale keuzehulp voor kinderondersteuning bij sport, gericht op helderheid, vertrouwen en gebruiksgemak op mobiel.
+summary: De keuzehulp begeleidt inwoners goed naar de plek waar ze hun aanvraag met vertrouwen kunnen beginnen. De keuzehulp is makkelijk te gebruiken. Er hoeft niet tot nauwelijks nagedacht te worden bij het beantwoorden van de vragen. 15 van de 17 inwoners kwamen snel en zonder problemen op de plek waar ze volgens het scenario moesten zijn. 12 van de 17 inwoners hadden er vertrouwen in dat ze kort na hun aanvraag het geld zouden ontvangen waarmee ze het lidmaatschap, de sportlessen en/of sportspullen zouden kunnen betalen. Voer voor livegang de verbeteringen uit dit onderzoek en het vorige onderzoek door om het voor de inwoner gebruiksvriendelijker en duidelijker te maken.
```

Comment:
De link in het originele stuk kon ik hier niet behouden. Omdat het bij het stukje **Doel van het onderzoek** nog een keer voorkomt heb ik het uit de samenvatting weg gelaten